README.md

```diff
@@ -73,6 +73,12 @@ Export a `COUCH_URL` env variable so sentinel knows what database to use. e.g.
 export COUCH_URL='http://admin:pass@localhost:5984/medic'
 ```
 
+If you are using CouchDB2.0 you need to also provide your node name. e.g.
+
+```
+export COUCH_NODE_NAME=couchdb@localhost node server.js
```

Comment:
It's the default that I have. I didn't change it to that. I presume this is what you have as well? If not we can change it to whatever